src/data/orgs/sisterslab-istanbul.json

```diff
@@ -0,0 +1,34 @@
+{
+  "image": "https://pbs.twimg.com/profile_images/993115542117548040/uAhtJo87_400x400.jpg",
+  "name": "Sisterslab",
+  "country": "turkey",
+  "city": "istanbul",
+  "topics": ["STEAM"],
+  "mainLink": "https://linktr.ee/sisterslabb",
+  "secondaryLinks": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/SistersLab"
+    },
+    {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/sisterslabb/"
+    },
+    {
+      "name": "Kommunity",
+      "url": "https://kommunity.com/sisterslabdeyiz"
+    },
+    {
+      "name": "Linktr",
+      "url": "https://linktr.ee/sisterslabb"
+    },
```

Comment:
Nit: Does it make sense to add the same link both as the `mainLink` and in the list of `secondaryLinks` @etelsv?

Reply:
Hmm, you are right. Removing the secondary one.